flutter-for-ios.md

```diff
@@ -27,3 +27,1085 @@ that are most relevant to your needs.
+
+## Where do I store strings? How do I handle localization?
+
+Unlike iOS, which has the `Localizable.strings` file, Flutter does not
+currently have a dedicated system for handling strings. At the moment, the
+best practice is to declare your copy text in a class as static fields and
+access them from there. For example:
+
+<!-- skip -->
+{% prettify dart %}
+class Strings {
+  static String welcomeMessage = "Welcome To Flutter";
+}
+{% endprettify %}
+
+You can access your strings as such:
+
+<!-- skip -->
+{% prettify dart %}
+new Text(Strings.welcomeMessage)
+{% endprettify %}
+
+
+While there's no functionality built into Flutter to handle
+internationalization or localization, Flutter developers are encouraged to use
+the [intl package](https://pub.dartlang.org/packages/intl) for those needs.
+
+Note that before Flutter 1.0 beta 2, assets defined in Flutter were not
+accessible from the native side, and vice versa, native assets and resources
+weren’t available from Flutter as they lived in separate folders.
+
+## What is the equivalent of Cocoapods? How do I add dependencies?
+
+In iOS, you add dependencies by adding to your `Podfile`. Flutter uses Dart’s
+own build system and the Pub package manager to handle dependencies. They then
+delegate the building of the native Android and iOS wrapper apps to the
+respective build systems. While there is a Podfile in the iOS folder in your
+Flutter project, you should only use this if you are adding native
+dependencies needed for per-platform integration. In general, you can use
+`pubspec.yaml` to declare external dependencies in Flutter. A good place to
+find great packages for Flutter is
+[Pub](https://pub.dartlang.org/flutter/packages/).
+
+# ViewControllers
+
+## What is the equivalent to `ViewController`s in Flutter?
+
+In iOS, a `ViewController` represents a portion of user interface, most
+commonly used for a screen or section. These can be composed together to build
+complex user interfaces, and help scale your application UI. In Flutter this
+concept falls under the umbrella of `Widget`s. As mentioned in the Navigation
+section, screens in Flutter are represented by `Widget`s since everything is a
+widget in Flutter. You use a `Navigator` to move between different `Route`s
+which represent different screens or pages, or maybe just different states or
+renderings of the same data.
+
+## How do I listen to iOS lifecycle events?
+
+In iOS, you can override methods to the `ViewController` to capture lifecycle
+methods for the view itself, or register lifecycle callbacks in the
+`AppDelegate`. In Flutter you have neither concept, but you can instead listen
+to lifecycle events by hooking into the `WidgetsBinding` observer and
+listening to the `didChangeAppLifecycleState()` change event.
+
+The lifecycle events you can observe are:
+
+* `inactive` — The application is in an inactive state and is not receiving
+user   input. This event only works on iOS, as there is no equivalent event to
```

Comment:
Spacing

Reply:
Fixed on #993, not sure what happened there.